views/explorer.tmpl

```diff
@@ -84,26 +84,26 @@
                 <thead>
                     <tr class="text-secondary">
                         <th class="text-left pl-0">Height</th>
-                        <th class="text-center d-none d-sm-table-cell">
+                        <th class="text-right d-none d-sm-table-cell">
                             <span class="d-none d-md-inline">Transactions</span>
                             <span class="d-md-none position-relative" data-tooltip="regular transactions">R</span>
                         </th>
-                        <th class="text-center d-none d-sm-table-cell">
+                        <th class="text-right d-none d-sm-table-cell">
                             <span class="d-none d-md-inline">Votes</span>
                             <span class="d-md-none position-relative" data-tooltip="votes">V</span>
                         </th>
-                        <th class="text-center d-none d-sm-table-cell">
+                        <th class="text-right d-none d-sm-table-cell">
                             <span class="d-none d-md-inline">Tickets</span>
                             <span class="d-md-none position-relative" data-tooltip="tickets">T</span>
                         </th>
-                        <th class="text-center d-none d-sm-table-cell">
+                        <th class="text-right d-none d-sm-table-cell">
                             <span class="d-none d-md-inline">Revocations</span>
                             <span class="d-md-none position-relative" data-tooltip="revocations">X</span>
                         </th>
-                        <th class="text-center d-table-cell d-sm-none">Txns</th>
-                        <th class="text-center">DCR</th>
-                        <th class="text-center">Size</th>
-                        <th class="text-center d-none d-sm-table-cell">
+                        <th class="text-right d-table-cell d-sm-none">Txns</th>
+                        <th class="text-right">DCR</th>
+                        <th class="text-right">Size</th>
+                        <th class="text-right d-none d-sm-table-cell">
```

Comment:
I'm not sure this is better either.

Before:

![image](https://user-images.githubusercontent.com/9373513/78581043-effab080-77f8-11ea-9169-aaa7eccba751.png)

This PR:

![image](https://user-images.githubusercontent.com/9373513/78581082-fd179f80-77f8-11ea-9703-4463b923cb28.png)

Note the "DCR" and "Size" header positions compared to the data cells below them.  "Version" is similarly off a bit.  And this seems to create a lot of empty space in the "Transactions" column that should be dealt with appropriately.

I might be OK with right aligning the numberic data **if** the column headers remain center aligned.